src/wallet/wallet.cpp

```diff
@@ -1916,47 +1918,29 @@ bool CWalletTx::IsEquivalentTo(const CWalletTx& _tx) const
         return CTransaction(tx1) == CTransaction(tx2);
 }
 
-// Rebroadcast transactions from the wallet. We do this on a random timer
-// to slightly obfuscate which transactions come from our wallet.
-//
-// Ideally, we'd only resend transactions that we think should have been
-// mined in the most recent block. Any transaction that wasn't in the top
-// blockweight of transactions in the mempool shouldn't have been mined,
-// and so is probably just sitting in the mempool waiting to be confirmed.
-// Rebroadcasting does nothing to speed up confirmation and only damages
-// privacy.
+// Once a day, resbumit all wallet transactions to the node,
```

Comment:
Not sure I follow. This method would be a no-op if the transaction is in your mempool. If its not in your mempool, it would call through to `BroadcastTransaction` with relay=false, so it would go through ATMP, but not be immediately relayed. Another `INV` will only be sent if the transaction is selected for rebroadcast. 

I’m sure there’s some small privacy leak here based on timing of re-entrance to mempool. Haven’t fully thought it through yet. But should be much less than before? What two updates are you referring to?